scripts/runtime/VibeRuntime.Common.ps1

```
@@ -1419,6 +1443,1663 @@ function New-VibeRuntimeSummaryDeliveryAcceptanceProjection {
     }
 }
 
+function Read-VibeJsonArtifactIfExists {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Read-VibeJsonArtifactIfExists' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Read-VibeJsonArtifactIfExists'

(PSUseSingularNouns)